build-scripts/config_common.cmake

```diff
@@ -201,6 +201,11 @@ elseif (WAMR_BUILD_LIBC_WASI EQUAL 1)
 else ()
   message ("     Libc WASI disabled")
 endif ()
+if (WAMR_BUILD_ENABLE_LIBC_WASI_SYNC_FLAGS EQUAL 1)
+  message ("     Libc WASI file descriptor sync flags enabled")
+else()
+  message ("     Libc WASI file descriptor sync flags disabled")
```

Comment:
Suggest not outputting the message of else since there are a lot messages already, no need to output some not so important messages. How about only outputting it when enabled?

Reply:
Sure 👍